Source/EasyNetQ/ConnectionConfiguration.cs

```diff
@@ -90,21 +92,29 @@ private void SetDefaultClientProperties(IDictionary<string, object> clientProper
             var hostname = Environment.MachineName;
             var product = Product ?? applicationName;
             var platform = Platform ?? hostname;
+            var name = Name ?? applicationName;
```

Comment:
@mantasaudickas You can use the prerelease version of EasynetQ on nuget. It's quite stable, I think we should release it.